### ISSUE TITLE
Bust not properly attributed

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#517]
```
Dealing #155190: 1500 3000 (500) Stud (High/Low Eight Qualifier)
Everyone antes 300
1     memyself      6550 300  [  ] [  ] 7h
2 B  V royalflush♠️ 10600 300  [  ] [  ] Qh
3     ☕ deadhead ☕  31500 300  [  ] [  ] 8c
4     🙈pokerchimp🙉  50050 300  [  ] [  ] Qc
5   >V fryday       100 300  [  ] [  ] 3s
6     JPMassar      850 300  [  ] [  ] Tc
               My cards are: [Kd] [9d] 8c
fryday brings-in for 100 and is all-in due to on vacation
fryday folds due to on vacation
When it's your turn to act, you'll try to call 0
JPMassar raises 350 to 850 and is all-in
memyself calls
royalflush♠️ folds due to being a no-show
☕ deadhead ☕ folds
🙈pokerchimp🙉 folds
1     memyself      5700 1150  6h 4h 7h
2 B  [royalflush♠️] 10600[ 300] 
3    [☕ deadhead ☕] 31500[ 300] 
4    [🙈pokerchimp🙉] 50050[ 300] 
5    [fryday]       0[ 400] 
6     JPMassar      0 1150  3d 4d Tc
1     memyself      5700 1150  6h 4h 7h 3c
2 B  [royalflush♠️] 10600[ 300] 
3    [☕ deadhead ☕] 31500[ 300] 
4    [🙈pokerchimp🙉] 50050[ 300] 
5    [fryday]       0[ 400] 
6     JPMassar      0 1150  3d 4d Tc 5s
1     memyself      5700 1150  6h 4h 7h 3c 3h
2 B  [royalflush♠️] 10600[ 300] 
3    [☕ deadhead ☕] 31500[ 300] 
4    [🙈pokerchimp🙉] 50050[ 300] 
5    [fryday]       0[ 400] 
6     JPMassar      0 1150  3d 4d Tc 5s 2c
1     memyself      5700 1150  6h 4h 7h 3c 3h Kh
2 B  [royalflush♠️] 10600[ 300] 
3    [☕ deadhead ☕] 31500[ 300] 
4    [🙈pokerchimp🙉] 50050[ 300] 
5    [fryday]       0[ 400] 
6     JPMassar      0 1150  3d 4d Tc 5s 2c Jc
1     memyself      5700 1150  6h 4h 7h 3c 3h Kh 9c
2 B  [royalflush♠️] 10600[ 300] 
3    [☕ deadhead ☕] 31500[ 300] 
4    [🙈pokerchimp🙉] 50050[ 300] 
5    [fryday]       0[ 400] 
6     JPMassar      0 1150  3d 4d Tc 5s 2c Jc 4s
memyself  6h 4h 9c
    High  King High Flush   Kh 7h 6h 4h 3h
     Low  none
JPMassar  3d 4d 4s
    High  Pair of Fours     4d 4s Jc Tc 5s
     Low  none
memyself  wins 3600 for High

JPMassar was busted by memyself in 11th place
fryday was busted by warren in 12th place
memyself: gg
☕ deadhead ☕: oops!
☕ deadhead ☕: Warren reappeared.
```
I'm currently tagging this as `easy` because it's almost undoubtedly due to recent mods.

**Comment by ctm** [comment]
FWIW, when I implemented #239, I found and fixed a similar issue before deploying. I had to allow for the fact that even when there were no callers I had to update the `seat_bounties` `HashMap`. What happened yesterday, however, is when there is a caller, so most likely there's just an instance where the current attribution code in the "there are callers" path incorrectly handles the case of a player who is all-in yet folded.

I still think this will be easy to track down and fix. I just haven't had time to look into it yet.
